new-website/deepchem/components/Home/BouncingAtoms/BouncingAtoms.styles.css

```diff
@@ -0,0 +1,9 @@
+html, body {
+  margin: 0;
+  padding: 0;
+}
+canvas {
+  display: block;
+  background: rgb(0,78,152);
+  background: linear-gradient(35deg, rgba(0,78,152,1) 0%, rgba(58,110,165,1) 34%, rgba(255,103,0,1) 100%);
```

Comment:
Can we add comments saying what these colors are?

Reply:
Added